droid-hal-device.inc

```diff
@@ -1176,8 +1203,13 @@ done
 %files kernel-dtbo -f dtbo.files
 %defattr(644,root,root,-)
 
+%if 0%{!?have_custom_vendor_boot:1}
 %if 0%{?enable_vendor_boot_update:1}
 %files vendor_boot
 /boot/vendor_boot.img
 %defattr(644,root,root,-)
 %endif
+%else
+%files kernel-modules-vendor_boot -f kernel-modules-vendor_boot.files
+%defattr(644,root,root,-)
```

Comment:
Do we need defattr here?

Reply:
I'll do a thorough cleanup of the spec some other time.